dll/win32/msv1_0/msv1_0.c

```diff
@@ -102,18 +102,25 @@ BuildInteractiveProfileBuffer(IN PLSA_CLIENT_REQUEST ClientRequest,
     PVOID ClientBaseAddress = NULL;
     LPWSTR Ptr;
     ULONG BufferLength;
+    SIZE_T ComputerNameLength;
     NTSTATUS Status = STATUS_SUCCESS;
 
     *ProfileBuffer = NULL;
     *ProfileBufferLength = 0;
 
+    ComputerNameLength = wcslen(ComputerName);
```

Comment:
`if (!NT_SUCCESS(RtlULongPtrToUShort(wcslen(ComputerName), &ComputerNameLength))))`
Please avoid all those cast that come afterwards.